### DISCUSSION TITLE
.link 文件在recbole框架中是如何使用的呢

### DISCUSSION THREAD

**Discussion by brokenvvorld** [RUCAIBox/RecBole#1348] [Q&A] [answered]
如题所示，之前跑过两个数据集的实验分别是book-crossing和ml-1m，他们的link文件其实无所谓，因为entity_id对应的item_id，但是在lastfm数据集中，这俩变得不一样了，所以我尝试去看了看recbole其他kg模型的实现，试图找到是如何使用.link文件。
但找了一圈却没找到如何使用，kg模型中没有用到.link文件得到的dataset.item2entity 或者 dataset.entity2item。
所以想问一下在recbole框架中，该文件是如何被处理使用的呢?

**Comment by Ethan-TZ** [comment, marked as answer]
@Kongxiangzhouye 您好，感谢您的关注！
RecBole框架中加载.link文件得到entity_id与item_id的对应关系，即变量item2entity与entity2item。而这两个变量的作用是将entity_id和item_id映射到同一internal id空间，其实现参考`_merge_item_and_entity`方法。

**Reply by brokenvvorld** [reply]
了解，原来是已经实现好了。感谢您的耐心回复!
